examples/networking/subnet_route/base-port.tf

```diff
@@ -0,0 +1 @@
+../port/main-port-for-subnet-route.tf
```

Comment:
place the enture example here, do not link it

Reply:
we do not need port for subnet_route i deleted it